proto/cometbft/state/v2/types.proto

```diff
@@ -0,0 +1,73 @@
+syntax = "proto3";
+package cometbft.state.v2;
+
+import "cometbft/abci/v2/types.proto";
+import "cometbft/state/v1/types.proto";
+import "cometbft/types/v1/types.proto";
+import "cometbft/types/v1/validator.proto";
+import "cometbft/types/v2/params.proto";
+import "gogoproto/gogo.proto";
+import "google/protobuf/timestamp.proto";
+
+option go_package = "github.com/cometbft/cometbft/api/cometbft/state/v2";
+
+// ABCIResponses retains the responses
+// of the various ABCI calls during block processing.
+// It is persisted to disk for each height before calling Commit.
+message ABCIResponses {
+  repeated cometbft.abci.v2.ResponseDeliverTx deliver_txs = 1;
+  cometbft.abci.v2.ResponseEndBlock end_block = 2;
+  cometbft.abci.v2.ResponseBeginBlock begin_block = 3;
```

Comment:
These might disappear (see my comments above), which may cause this one to disappear as well